agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -283,11 +291,14 @@ func TestTelemetry(t *testing.T) {
 	params.EndTime = aws.Time(RoundTimeUp(time.Now(), time.Minute).UTC())
 	params.StartTime = aws.Time((*params.EndTime).Add(-waitMetricsInCloudwatchDuration).UTC())
 	params.MetricName = aws.String("CPUUtilization")
-	err = VerifyMetrics(cwclient, params, false)
+	err, metrics := VerifyMetrics(cwclient, params, false)
 	assert.NoError(t, err, "Task is running, verify metrics for CPU utilization failed")
+	// Also verify the cpu usage is around 25%
+	assert.True(t, *metrics.Average < 0.3)
```

Comment:
I think you can use [`assert.InDelta`](https://godoc.org/github.com/stretchr/testify/assert#InDelta).

```go
assert.InDelta(t, 0.25, *metrics.Average, 0.05)
```